src/lib/cpu_info/libcpu_info.h

```diff
@@ -1,7 +1,5 @@
-#ifndef HAL_CPU_INFO_H_
```

Comment:
Please add header guard to avoid double `#include`

Reply:
This is outdated, `hal/cpu_info.h` is not needed anymore.

Reply:
Added in `lib/cpu_info/libcpu_info.h`.